src/test/java/org/springframework/data/couchbase/domain/Airport.java

```diff
@@ -48,8 +52,16 @@ public class Airport extends ComparableEntity {
 	@Max(2) long size;
 	private long someNumber;
 
+	public AirportJsonValuedObject jvObject;
+
+	public Airport() {
+		this.jvObject = new AirportJsonValuedObject();
+		this.jvObject.theValue = "first";
+	}
+
 	@PersistenceConstructor
 	public Airport(String key, String iata, String icao) {
+		this();
```

Comment:
NIT: A more reliable way to share init code between constructors is to put the code in a non-static init block.

```
{
  this.jvObject = new AirportJsonValuedObject();
  this.jvObject.theValue = "first";
}
```

https://stackoverflow.com/questions/3987428/what-is-an-initialization-block

Reply:
yes